components/compliance-service/ingest/ingestic/ingestic.go

```diff
@@ -43,21 +43,30 @@ func (backend *ESClient) addDataToIndexWithID(ctx context.Context,
 }
 
 // This method will support adding a document with a specified id
-func (backend *ESClient) upsertComplianceRunInfo(ctx context.Context, mapping mappings.Mapping, id string, runDateTime time.Time) error {
-	runDateTimeAsString := runDateTime.Format(time.RFC3339)
+func (backend *ESClient) upsertComplianceRunInfo(ctx context.Context, mapping mappings.Mapping, runInfo relaxting.ESComplianceRunInfo, runDateTime time.Time) error {
 
-	script := elastic.NewScript("ctx._source.last_run = params.rundate").Param("rundate", runDateTimeAsString)
+	firstRunInfo, err := backend.getDocFromNodeRunInfoFromNodeId(ctx, runInfo.NodeID, mapping.Index)
```

Comment:
We are doing this additional call to Open Search to get the first run information. Can we do it without making this call?